week-2/Python/prabhjot-singh.py

```diff
@@ -0,0 +1,64 @@
+
+from itertools import permutations
+
+# Given a string, find the longest substring which is palindrome. 
+# For example, if the given string is "ababad", the output should be "ababa".
+def longestPalindromeSubstring(string):
+  longest = ''
+
+  # O(n^2), n = length of string
```

Comment:
Slight mistake while calculating time complexity. I didn't take string reversing on line 20 into account. Time complexity should be O(n^3).